local_minio/minio_install.sh

```diff
@@ -1,69 +1,76 @@
 #!/bin/bash
+source "utils/parameters.sh"
 
+# IPS_ARR=(
 
-IPS_ARR=(
-    192.168.89.6
-    192.168.89.13
-    192.168.89.20
-    192.168.89.5
-    192.168.89.17
-    192.168.89.10
-    192.168.88.253
-    192.168.89.21
-    192.168.89.0
-    192.168.89.22
-    192.168.89.4
-    192.168.89.8
-    192.168.89.14
-    192.168.89.9
-    192.168.89.2
-    192.168.88.254
-    192.168.89.1
-    192.168.89.24
-    192.168.89.29
-    192.168.89.103
-    192.168.88.252
-    192.168.89.16
-    192.168.89.11
-    192.168.89.18
-    192.168.89.7
-    192.168.89.3
-    192.168.89.19
-    192.168.89.15
-    192.168.89.64
-    192.168.89.65
-    192.168.89.104
-    192.168.89.91
-)
-ACCESS_KEY=access_key
-SECRET_KEY=secret_key
-GROUPID=jimmy-minio-server
+GROUPID=minio-server
 
 function do_install {
-  do_uninstall
+  uninstall_only
 
-  for ((i=0;i<${#IPS_ARR[@]};++i)); do
-    echo "${IPS_ARR[i]} $GROUPID-$i" >> /etc/hosts
-  done
+  # cachengo-cli updateInstallStatus $APPID "Installing"
+  local HOSTS_ARR
+  array_from_json_list HOSTS_ARR "$HOSTNAMES"
+  array_len=$((${#HOSTS_ARR[@]}-1 ))
+  export MINIO_ACCESS_KEY=$ACCESS_KEY
+  export MINIO_SECRET_KEY=$SECRET_KEY
 
-  echo "Total: $i"
+  apt install -y avahi-utils
+  apt install -y python3
+  apt install -y curl
+
+  #output hostnames to use with lookup service to a common file
+  echo $HOSTNAMES > /usr/bin/minio_lookup_hostnames.json
```

Comment:
This seems messy. I think it would be better to add that to a variable in the environment on the service file.